### PR TITLE
add --fix argument to ESLint

### PR DESCRIPTION
**Pull request by Obinnabii**
This should make ESLint fix some of the stuff it flags. things to note about this PR include;
- I'm using the [--fix](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) ESLint command.
- I switched to the [Run ESLint Action](https://github.com/marketplace/actions/run-eslint#prerequisites) GitHub Action
- The changes from the ESLint Action are committed using [git-auto-commit action](https://github.com/stefanzweifel/git-auto-commit-action). 
- The [previous linter](https://github.com/marketplace/actions/eslint-action-eslintrc) was preserved and used to annotate ESLint errors that cannot be automatically fixed.
- I added a Personal Access token to allow GitHub Actions to run recursively on any push that the linter makes. The token is [PAT_TOKEN secret](https://github.com/googleinterns/step22-2020/settings/secrets) for this repo.
  
__Note:__ _The Linter ran on this PR. The changes that I directly made are in the ```formatting.yaml``` and ```.eslintrc``` files_

__TODO:__ add comments to the README

### CODE REVIEW

**Comment by Obinnabii** [comment]
@Richie78321 is working on fixing the last two linter errors on PR #72

File: .github/workflows/formatting.yaml [hunk 1]
```diff
@@ -18,11 +18,24 @@ jobs:
...
+        token: ${{secrets.PAT_TOKEN}}
+    - name: Install modules
+      run: npm install
+    - name: Run ESLint
+      continue-on-error: true
+      run: npx eslint . --ext .js,.jsx,.ts,.tsx --fix
+    - name: Commit ESLint changes
+      uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: apply ESLint changes
```
Comment (billyeh) [inline .github/workflows/formatting.yaml:36]:
```suggestion
        commit_message: Apply ESLint changes.
```
nit

File: src/main/webapp/dashboard.js [hunk 2]
```diff
@@ -196,14 +196,17 @@ function createCardTitle(titleText) {
 /**
  * Return a card with information about a given @param mentee object and
  * a link to send @param mentee a star
+ *
+ *
+ *
```
Comment (billyeh) [inline src/main/webapp/dashboard.js:201]:
```suggestion
```

**Comment by billyeh** [comment]
Really awesome job getting this working!

**Pull request merged**